frontend/src/v5/services/realtime/ticket.events.ts

```diff
@@ -22,6 +22,7 @@ import { getMeshIDsByQuery } from '@/v4/services/api';
 import { meshObjectsToV5GroupNode } from '@/v5/helpers/viewpoint.helpers';
 import { subscribeToRoomEvent } from './realtime.service';
 import { TicketsActionsDispatchers } from '../actionsDispatchers';
+import { fetchTicketGroup } from '../api/tickets';
```

Comment:
Is this a bug or a quirk of the model @carmenfan ?
If I select the floor when it saves the group I get all meshes

![selects all](https://github.com/3drepo/3drepo.io/assets/42032433/fd3fc406-1b1b-47c6-8510-5f25c92e853f)

Reply:
I had the same issue. It's a problem specific to this model apparently 
![image](https://github.com/3drepo/3drepo.io/assets/43547703/8cd8cf97-bdf2-417b-9b82-5fc5743fd363)

Reply:
as @Amantini1997  said.

Use something in the building